server/protocols/jmap-rfc-8621-integration-tests/jmap-rfc-8621-integration-tests-common/src/main/scala/org/apache/james/jmap/rfc8621/contract/EchoMethodContract.scala

```diff
@@ -147,14 +117,31 @@ trait EchoMethodContract {
         .body()
         .asString()
 
-    assertThatJson(response).isEqualTo(RESPONSE_OBJECT)
+    assertThatJson(response).isEqualTo(Fixture.ECHO_RESPONSE_OBJECT)
+  }
+
+  @Test
+  @Tag(CategoryTags.BASIC_FEATURE)
+  def echoMethodShouldRespond401WithRFC8621VersionWhenWrongAuthentication(): Unit = {
+    val authorizationValue: String =
+      s"Basic ${toBase64("alice@@domain.tld:bobpassword")}"
```

Comment:
Please do this at least with a valid email address

Reply:
We should not puyt authentication tests in echo method tests

Reply:
we already have a valid email addess, this case for the wrong one

Reply:
I will move all test about Authentication to AuthenticationContract